components/identities/actions/transferFormModal.tsx

```diff
@@ -89,20 +89,48 @@ const TransferFormModal: FunctionComponent<TransferFormModalProps> = ({
 
   const modalContent = (
     <>
-      <p className="mt-5">
+      <p className="mt-5 md:text-sm text-xs text-center text-[#8C8989] md:leading-6">
         An Identity is an NFT that everyone can mint for free that permits
         linking different types of data to it (Social Media, stark domain ...).
         This form enables you to send this identity to another wallet.
       </p>
-      <div className="mt-5 flex flex-col justify-center">
+      <div className="mt-5  flex flex-col justify-center  w-full">
         <TextField
```

Comment:
Instead, use AdvancedTextField from components/UI/advancedTextField.tsx, which is exactly like the figma one. It'll be easier for you to have exactly the same look as figma for that part, and it's always better to reuse components than rewriting duplicated code!